ui/src/main.css

```diff
@@ -55,7 +55,8 @@ input[type='color']:focus,
   z-index: 10000;
 }
 
-.modal-dialog {
```

Comment:
The Add Sample modal doesn't need to be that big. Instead of overriding these styles locally, I introduce an attribute to simply opt out of them.